storage/Devices/BcacheImpl.cc

```diff
@@ -45,25 +46,35 @@ namespace storage
     using namespace std;
 
 
-    // TODO rename after delete (in memory)
     // TODO handle attach and detach
     // TODO synchronization after actions
-    // TODO cache-mode
 
 
     const char* DeviceTraits<Bcache>::classname = "Bcache";
 
+    const vector<string> EnumTraits<CacheMode>::names({
+	"writethrough", "writeback", "writearound", "none"
+    });
 
     Bcache::Impl::Impl(const string& name)
-	: Partitionable::Impl(name)
+	: Partitionable::Impl(name), cache_mode(CacheMode::WRITETHROUGH),
+          sequential_cutoff(8 * MiB), writeback_percent(10) // bcache defaults
     {
 	update_sysfs_name_and_path();
     }
 
 
     Bcache::Impl::Impl(const xmlNode* node)
-	: Partitionable::Impl(node)
+	: Partitionable::Impl(node), cache_mode(CacheMode::WRITETHROUGH),
+          sequential_cutoff(8 * MiB), writeback_percent(10) // bcache defaults
     {
+	string tmp;
+
+	cache_mode = CacheMode::WRITETHROUGH;
+	if (getChildValue(node, "cache_mode", tmp))
```

Comment:
The convention is to use '-' instead of '_' in the xml files.

Reply:
ok, will change